### PR TITLE
Ft update login screen

### PR DESCRIPTION
**Pull request by bkandaris**
## Description
Cleaned up the CSS on the login screen (NOT Register...yet)
Fixed 'htmlFor' error by changing it to 'htmlfor' no camelCase
Made Image larger on 'Desktop Screen' 
Fixed hover effects to make them look better
Fixed 'Link' issue when creating a store by changing the <button> to a <p>

## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings

## Link to Trello card
https://trello.com/c/d4BSvctZ

### CODE REVIEW

File: src/components/login.js
Comment (Cireimu) [inline src/components/login.js]: Getting a massive red warning, just need you change htmlType to htmltype. Besides that though it looks great and ready to merge after someone else checks it out and approves it!
change htmlType to htmltype

**Review by Cireimu — DISMISSED** [review]

**Review by MatthewVaccaro — COMMENTED** [review]
Needs Fixing:
On mobile you have the old logo show up
If you click the 'Create Store Button' it doesn't navigate away and the error message pop

Would-Be Nice
The image was a bit larger on desktop (800px by 600px and use the CSS property `object-fit: cover`
logo feels weird changing size with the browser. Would recommend a single size
Hover colors feel odd. I can give you better ones if you're willing to make the change

**Comment by bkandaris** [comment]
> object-fit: cover

**Review by Cireimu — DISMISSED** [review]
Looks great, just pull from the develop branch and solve merge conflicts and I'll get it merged. Great work so far Ben!

**Comment by Cireimu** [comment]
Just need you to pull from develop and resolve merge conflicts and it will be good to go and get merged. Good work Ben!

**Review by Cireimu — APPROVED** [review]

**Review by MatthewVaccaro — APPROVED** [review]

**Pull request merged**
